source/extensions/filters/http/cache/http_cache.h

```diff
@@ -170,6 +170,16 @@ using LookupResultPtr = std::unique_ptr<LookupResult>;
 // TODO(toddmgreer): Ensure that stability guarantees above are accurate.
 size_t stableHashKey(const Key& key);
 
+// The metadata associated with a cached response.
+// TODO(yosrym93): This could be changed to a proto if a need arises.
+struct ResponseMetadata {
+  // The time at which a response was was most recently inserted, updated, or validated in this
+  // cache. This represents "response_time" in the age header calculations at:
+  // https://httpwg.org/specs/rfc7234.html#age.calculations
+  SystemTime response_time_;
+};
+using ResponseMetadataPtr = std::unique_ptr<ResponseMetadata>;
```

Comment:
not used?

Reply:
Nice catch, that's a remnant of a previous implementation. Removed.